integration/vendored_test.go

```diff
@@ -56,19 +53,26 @@ func testVendored(t *testing.T, context spec.G, it spec.S) {
 			Expect(err).NotTo(HaveOccurred())
 
 			image, _, err = pack.Build.
-				WithBuildpacks(nodeCachedURI, yarnCachedURI).
+				WithBuildpacks(
+					nodeOfflineURI,
+					yarnOfflineURI,
+					buildpackOfflineURI,
+					buildPlanURI,
+				).
 				WithNetwork("none").
 				Execute(name, source)
 			Expect(err).NotTo(HaveOccurred())
 
-			container, err = docker.Container.Run.Execute(image.ID)
+			container, err = docker.Container.Run.
+				WithCommand("ls -alR /layers/paketo-buildpacks_yarn-install/modules/node_modules").
```

Comment:
Can we avoid hardcoding `paketo-buildpacks_yarn-install` and instead derive the layer name from the buildpack.toml info?